### ISSUE TITLE
blackbox_shred_all_files throwing up directory error

### ISSUE THREAD

**Issue by ashwin1122** [StackExchange/blackbox#311]
`blackbox_shred_all_files` is throwing up the following error:

```
   $ blackbox_shred_all_files
========== FILES BEING SHREDDED:
environment: /c/Users/Ashwin/_blackbox_common.sh: No such file or directory
NOT FOUND: Nair/Desktop/blackbox/bin
========== DONE.
```
This maybe because the path is breaking at a whitespace but blackbox_postdeploy which references the same directory `C:/Users/Ashwin Nair/Desktop/blackbox/bin/` name is working fine.

**Comment by tlimoncelli** [comment]
Interesting problem!

Question: Do other blackbox commands work?  They all use the same way to find `_blackbox_common.sh`

As a temporary workaround, you can edit blackbox_shred_all_files?  Line 19 looks like:

    source "${0%/*}/_blackbox_common.sh"
 
change it to:

    source "/c/Users/Ashwin Nair/Desktop/blackbox/bin/_blackbox_common.sh"

**Comment by ashwin1122** [comment]
Yeah the other blackbox commands seem to be working fine. The temporary workaround is also throwing up the same error as earlier.

**Comment by tlimoncelli** [comment]
Huh. I'm kind of stumped.  All the scripts fine `_blackbox_common.sh` the same way.

What release are you using?  Is it possible to use a symlink or otherwise avoid having a space in the path?

**Comment by ashwin1122** [comment]
Using the latest available release ```v1.20200429```. I reinstalled the files to a filepath that doesn't have a space in it and it's working fine now.

**Comment by TomOnTime** [comment]
Glad to hear it!
